libs/ledger-live-common/src/manager/api.ts

```diff
@@ -421,9 +432,9 @@ const getAppsByHash: (hashes: string[]) => Promise<Array<App>> = makeLRUCache(
       throw new NetworkDown("");
     }
 
-    return data.map(mapApplicationV2ToApp);
+    return data.map(appV2 => (appV2 ? mapApplicationV2ToApp(appV2) : null));
   },
-  hashes => String(hashes),
+  hashes => `${getEnv("MANAGER_API_BASE")}_${String(hashes)}`,
```

Comment:
Not big: I did not know what `String(hashes)` was going to look like, i think a `hashes.join("_")` would be more explicit

Reply:
yep I can do that